pkg/csi/service/common/authmanager.go

```diff
@@ -160,13 +181,13 @@ func ComputeDatastoreMapForBlockVolumes(authManager *AuthManager, authCheckInter
 	}
 }
 
-// ComputeDatastoreMapForFileVolumes refreshes DatastoreMapForFileVolumes periodically
-func ComputeDatastoreMapForFileVolumes(authManager *AuthManager, authCheckInterval int) {
+// ComputeDatastoreMapsForFileVolumes refreshes DatastoreMapForFileVolumes & clusterToFSDatastoreURLMap periodically
+func ComputeDatastoreMapsForFileVolumes(authManager *AuthManager, authCheckInterval int) {
```

Comment:
Since we're now computing two maps, making it plural to `Maps` seems logical. Also both the maps are related to datastores - one is `dsURL` to `dsInfo` map, the other one is cluster to list of corresponding datastores.

 > ComputeDatastoreMapForFileVolumes is better.
 
This doesn't imply we're computing multiple maps. Or we can just call it `ComputeMapsForFileVolumes`. Let me know if that makes sense.